squash/supervisord.conf

```diff
@@ -0,0 +1,26 @@
+[supervisord]
+nodaemon=true                                 ; run supervisor in foreground otherwise docker run exits
+
+[program:mysql]
+user=root
+command=/usr/bin/mysqld_safe
+stdout_logfile=/var/log/supervisor/mysql.log
+redirect_stderr=true                          ; Save stderr in the same log
+autostart=true
+
+[program:django]
+user=root
+; by default django listens to the "image localhost", since we are forwarding port 8000 to
+; the "external host" we have to make it listen to all interfaces
+; --noreload is needed to pass the control to supervisor
+command=/usr/local/bin/python /usr/src/app/squash/manage.py runserver 0.0.0.0:8000 --noreload
+stdout_logfile=/var/log/supervisor/django.log
+redirect_stderr=true
+autostart=true
+
+[program:bokeh]
+user=root
```

Comment:
It is generally a good idea not to run any services as the root user, even in a container.